docker/dev/install_odb.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+opt=${1:-"all"}
+
+wget https://raw.githubusercontent.com/Ericsson/CodeCompass/master/scripts/install_latest_build2.sh
+sh install_latest_build2.sh "/build2_install"
+export PATH=/build2_install/bin:$PATH
+# Configuring the build
+mkdir /odb_build
+cd /odb_build
+bpkg create --quiet --jobs $(nproc) cc \
+  config.cxx=g++ \
+  config.cc.coptions=-O3 \
+  config.bin.rpath=/odb_install/lib \
+  config.install.root=/odb_install
+# Getting the source
+bpkg add https://pkg.cppget.org/1/beta --trust-yes
+bpkg fetch --trust-yes
+# Building odb
+bpkg build libodb --yes
+case $opt in
+  "sqlite")
+    bpkg build libodb-sqlite --yes
+  ;;
+  "pgsql")
+    bpkg build libodb-pgsql --yes
+  ;;
+  *)
+    bpkg build odb --yes
+    bpkg build libodb-sqlite --yes
+    bpkg build libodb-pgsql --yes
+  ;;
+esac
+bpkg install --all --recursive
+# Copy to /usr
+cp -rn /odb_install/* /usr
```

Comment:
Conventionally the `/usr` folder is for files installed through the package manager. Locally compiled programs should go to `/usr/local`.

Therefore, simply `/usr/local` should be given as the install path and no copying is needed.

Reply:
We have discussed on the weekly meeting, that this could also be a valid approach. However, I have further doubts, as the install folder could also contain symlinks. If those are not relative, but absolute, copying them would result in invalid links. Therefore installing directly into `/usr/local/` would still be preferred.